src/components/Profile/components/VictoryPromise/components/GraphDisplay.tsx

```diff
@@ -18,60 +19,57 @@ const GraphDisplay = ({ scores }: Props) => {
    */
   const emptySliceValue = minimumScore * 0.6;
 
+  Chart.register(RadialLinearScale, ArcElement, Tooltip, Legend);
+  Chart.defaults.plugins.tooltip.callbacks.label = () => '';
+  Chart.defaults.plugins.tooltip.titleMarginBottom = 0;
+
   Object.entries(scores).forEach((score) => {
     const [key, value] = score as [VictoryPromiseCategory, number];
     const index = GraphOrder[key];
     const colorHex = (Colors[key].match(/#[A-Fa-f0-9]{6,8}/) || [''])[0];
     labels[index] = toTitleCase(key, '_');
+
     if (value > 0) {
       colorHex == '' ? (colors[index] = '#000000') : (colors[index] = colorHex);
       data[index] = value;
+      labels[index] += ` : ${value}`;
     } else {
       colors[index] = light_gray;
       data[index] = emptySliceValue;
+      labels[index] += ` : 0`;
     }
   });
 
-  const datasets = [
-    {
-      data,
-      backgroundColor: colors,
-      borderWidth: 3,
+  const dataOBJ = {
+    labels: labels,
+    datasets: [
+      {
+        data: data,
+        backgroundColor: colors,
+        borderWidth: 3,
+        borderColor: '#fff',
+      },
+    ],
+  };
+
+  const options = {
+    responsive: true,
+    plugins: {
+      legend: {
+        display: false,
+      },
+    },
+    scales: {
+      r: {
+        display: false,
+      },
+    },
+    layout: {
+      padding: 30,
```

Comment:
I think this is more padding than necessary:
```suggestion
      padding: 10,
```